tests/test_tokenization_common.py

```diff
@@ -2928,8 +2929,10 @@ def test_batch_encode_dynamic_overflowing(self):
                     returned_tensor = "pt"
                 elif is_tf_available():
                     returned_tensor = "tf"
-                else:
+                elif is_flax_available():
```

Comment:
This test was failing when no framework (PyTorch, TF, Flax) is installed. Returning instead of failing now.